### DISCUSSION TITLE
I am looking for a developer to add Web3 authorization on the site

### DISCUSSION THREAD

**Discussion by aecash** [Web3Auth/Web3Auth#1107] [Feature Requests]
This is the first step. And I will be grateful for any suggestions what else we can implement useful for our customers. I will be happy to discuss any proposals, even the most insane ones.

**Comment by tahirahmadin** [comment]
I can.
My TG: tahirahmadin

**Reply by aecash** [reply]
Thank you, Tahir, may you write me please. My TG iharkul
